src/JustSaying/AwsTools/MessageHandling/Compression/GzipMessageBodyCompressor.cs

```diff
@@ -0,0 +1,20 @@
+using System.IO.Compression;
+using System.Text;
+
+namespace JustSaying.AwsTools.MessageHandling.Compression;
+
+class GzipMessageBodyCompressor : IMessageBodyCompressor
```

Comment:
Unless you intended to make it public:

```suggestion
internal sealed class GzipMessageBodyCompressor : IMessageBodyCompressor
```